### PR TITLE
Modify team members, history and companies using Kedro

### PR DESCRIPTION
**Pull request by yetudada**
## Description
This PR specifically focuses on making modifications to the language used to describe Kedro, modifies our team member page and adds companies that are using Kedro.

## Checklist

- [x] Read the [contributing](https://github.com/quantumblacklabs/kedro/blob/master/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/quantumblacklabs/kedro/blob/master/RELEASE.md) file
- [ ] Added tests to cover my changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,7 +66,7 @@ Kedro is built upon our collective best-practice (and mistakes) trying to delive
 
 ## The humans behind Kedro
 
-Kedro is maintained by a [product team from QuantumBlack](https://kedro.readthedocs.io/en/stable/12_faq/01_faq.html) and a number of [contributors from across the world](https://github.com/quantumblacklabs/kedro/releases).
+Kedro is maintained by [a product team](https://kedro.readthedocs.io/en/stable/12_faq/01_faq.html) and a number of [contributors from across the world](https://github.com/quantumblacklabs/kedro/releases).
```
Comment (datajoely) [inline README.md:69]: Is the numbered link going to break?

**Review by datajoely — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (lorenabalan) [reply README.md:69]: It'll trigger a conflict with `develop` on `merge-master-to-develop`.

File: docs/source/12_faq/01_faq.md [hunk 2]
```diff
@@ -12,38 +13,39 @@ For the source code, take a look at the [Kedro repository on Github](https://git
...
+Andrew Mackay,
+[Antony Milne](https://github.com/AntonyMilneQB),
+[Gabriel Comym](https://github.com/GabrielComymQB),
 [Ivan Danov](https://github.com/idanov),
-[Richard Westenra](https://github.com/richardwestenra),
-[Lorena Balan](https://github.com/lorenabalan),
+[Joel Schwarzmann](https://github.com/datajoely),
 [Lim Hoang](https://github.com/limdauto),
-[Jo Stichbury](https://github.com/stichbury),
+[Lorena Bălan](https://github.com/lorenabalan),
```
Comment (lorenabalan) [inline docs/source/12_faq/01_faq.md:26]: 💯 ❤️

**Review by lorenabalan — APPROVED** [review]
Awesome! 🥇

**Pull request merged**